flare/otf.py

```diff
@@ -212,6 +222,20 @@ def run(self):
                     if (self.dft_count-1) < self.freeze_hyps:
                         self.train_gp()
 
+                    # Store DFT outputs in another folder if desired
+                    # specified in self.store_dft_output
+                    if self.store_dft_output is not None:
+                        dest = self.store_dft_output[1]
+                        target_files = self.store_dft_output[0]
+                        now = datetime.now()
+                        dt_string = now.strftime("%Y.%m.%d:%H:%M:%S:")
```

Comment:
This seems like a good way of labeling the output files. We should write the labeling convention in the docstring for the "run" method.